modules/check_in/app/models/check_in/v2/session.rb

```diff
@@ -109,9 +109,7 @@ def success_message
       # @return [Faraday::Response]
       #
       def client_error
-        body = { error: true, message: 'Invalid uuid, last4 or last name!' }
-
-        Faraday::Response.new(body: body, status: 400)
+        { error: true, message: 'Invalid uuid, last4 or last name!' }
```

Comment:
If this error passes through to be displayed to the user we should not mention UUID.

Reply:
Good idea. Will remove it in the next PR! 👍